src/legacy/core_plugins/tagcloud/public/tag_cloud_visualization.js

```diff
@@ -71,11 +71,11 @@ export class TagCloudVisualization {
 
   }
 
-  async render(data, status) {
+  async render(data, visParams, status) {
     if (!(status.resize || status.data || status.params)) return;
 
-    if (status.params || status.aggs) {
-      this._updateParams();
+    if (status.params || status.data) {
```

Comment:
`this._updateParams` and `this._updateData` have the same if condition now and can be merged into one function since they are not called from somewhere else.